### DISCUSSION TITLE
好麻烦！！！！

### DISCUSSION THREAD

**Discussion by ADAD33** [share121/inter-knot#2273] [其他]
绳网有移动端吗，每次在空洞看绳网都要用电脑，好麻烦<img src="https://img-static.mihoyo.com/communityweb/upload/5981af706fa790734aa32abfc2df996b.png" alt="比利 柔弱" width="75" >

**Comment by Kamisato100** [comment]
你可以手机端横屏用<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >

**Comment by xuXU347** [comment]
![Uploading 36cd425a-b6c0-4d3f-b36f-c04e622a3b0f.png…]()

**Comment by biliqide** [comment]
可以用via

**Comment by jianjue1** [comment]
油猴插件的话可以用x浏览器安装，参考链接
【【手机技巧】超简单！强大的电脑“油猴脚本”在安卓和苹果iOS安装方法-哔哩哔哩】 https://b23.tv/SIzW4s7
加速器未知，我全程科学上网（<img src="https://bbs-static.miyoushe.com/static/2023/11/16/b7b6eb7ebdc9f63ba6365e7e8d6b5fd7_5912345410510420967.png" alt="安比-啊···" width="75" >

**Comment by constant2854** [comment]
我用firefox ANDROID成功访问了

**Comment by 00zzxx** [comment]
有啊，手机上装个github就OK了<img src="https://img-static.mihoyo.com/communityweb/upload/54f6c5f6a4bf851877158b0000e636c5.png" alt="克拉拉 给你心心" width="75" >

**Comment by Brighttaz** [comment]
你不觉得电脑用起来·更有感觉么（）
